content/en/docs/architecture/release-gating.md

```diff
@@ -159,7 +159,7 @@ and will not swamp TRT or unreasonably prevent us from getting green payloads.
 
 However, in some scenarios, it may be possible to shortcut this process. We
 have recently gained the ability to run specific jobs an arbitrary number of
-times via [Gangway](https://github.com/stbenjam/gangway-cli). This allows us to
+times via [Gangway](https://github.com/stbenjam/gangway-cli), [document](https://docs.google.com/document/d/1x82wtDkEaWmqx627K2mpcWpIaP5PYsCoutenhMqkXVU/edit#heading=h.jy0krrkvq90t) on how to use Gangway. This allows us to
```

Comment:
Could we do a markdown page just for gangway? I think we prefer not to have private docs linked from docs.ci.

Reply:
@stbenjam gangway is not for public use and we don't want users to be coming and requesting access

Reply:
That's why the current documentation is appropriate, and I am closing this PR then :/